assignments/arrays.js

```diff
@@ -57,40 +57,88 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // Example for loop:
 // arr = [1,2,3,4];
-// for (i = 0; i < arr.length; i++) {
+// for (*let* i = 0; i < arr.length; i++) {
 //     arr[i]; // 1,2,3,4
 // }
+console.log();
+console.log("Arrays Begin");
+console.log("challenge 1");
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
 
+// index 32
+// {"id":33,"car_make":"Jeep","car_model":"Wrangler","car_year":2011}
 
+console.log();
+console.log("challenge 2");
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
+let lastCar = inventory[inventory.length - 1];
```

Comment:
You are the only person I've seen use the `lastCar` variable to store the object itself instead of the index. Good thinking.